### DISCUSSION TITLE
Request Slack access to #atlasmap on ASF

### DISCUSSION THREAD

**Discussion by mattrpav** [atlasmap/atlasmap#4626] [Project Infra]
Add a comment here with your email address to be granted access to the #atlasmap channel on ASF Slack server

**Comment by arockiaraj-eng** [comment]
Hi Matt,  

Please provide the slack access for the below mail 

Mail : arockiaraj1994@gmail.com

**Reply by mattrpav** [reply]
done

**Comment by SushmaReddyLoka** [comment]
Please provide the slack access for the below mail

Mail : sushmareddy.loka@gmail.com

**Reply by mattrpav** [reply]
done

**Comment by ShivamG0217** [comment]
can you give access to the below mail :

Email: shivam0217@gmail.com

**Reply by mattrpav** [reply]
done

**Comment by Arsh-2811** [comment]
Hi Matt,

Please provide the slack access for the below mail

Mail : [arsh.sharan2811@gmail.com](mailto:arsh.sharan2811@gmail.com)

**Reply by mattrpav** [reply]
done

**Comment by jeongm-in** [comment]
Hi, could you provide access for me as well? 
Mail:  sogogibeef@gmail.com

**Reply by mattrpav** [reply]
done

**Comment by fgapito** [comment]
Hi,

Please provide the slack access for the below mail

Mail: francesco.gapito@gmail.com

**Reply by mattrpav** [reply]
done

**Comment by dcolazin** [comment]
Hello, can you also give me access? Mail: dcolazin@gmail.com

**Reply by mattrpav** [reply]
done

**Comment by gowthamnaraganti-cloud** [comment]
Hello matt can you provide access to Mail `gowthamsai743@gmail.com`
